README.rst

```diff
@@ -29,8 +29,18 @@ Features
 
 Installation
 -------------
-See doc/install.txt in the source code archive.
-Python 2.7.2 or later is needed.
+
+You likely want to use 
+
+``pip install -e git+git@github.com:linkcheck/linkchecker.git#egg=linkchecker``
+
+or 
+
+``pip install --user -e git+git@github.com:linkcheck/linkchecker.git#egg=linkchecker``
+
+See doc/install.txt in the source code archive for partially outdated but more detailed instructions.
```

Comment:
how about fixing the documentation in install.txt instead? :)

Reply:
I just wanted to fix that python 3 documentation issue without doing a full refactoring of the documentation. The latter would extend my five minute pull request to digging into the intricacies of pip and reformatting lots of text, which I'd like to avoid.

Reply:
you don't have to edit install.txt - i just don't see how the above is necessary to fix #88 ...